### DISCUSSION TITLE
What's the future of typst?

### DISCUSSION THREAD

**Discussion by basings** [typst/typst#3033] [General]
I read that you are looking for business angels. Will this become proprietary at some point?

> We are currently operating on some Business Angel investment

https://github.com/sponsors/typst

**Comment by LaurenzV** [comment]
Not a founder, but they said that the compiler will always stay open source (anything other wouldn't make any sense). I don't know their details for monetization, but it will most likely involve monetizing their web app somehow.

**Reply by basings** [reply]
Thanks for your thoughts
